src/llmcompressor/modifiers/quantization/gptq/utils/gptq_wrapper.py

```diff
@@ -222,8 +227,22 @@ def compress(
             else:
                 W[:, i2:] -= w_err
 
-        logger.info("time %.2f" % (time.time() - tick))
-        logger.info("error %.2f" % torch.sum(Losses).item())
+        logger.log("METRIC", "time %.2f" % (time.time() - tick))
+        logger.log("METRIC", "error %.2f" % torch.sum(Losses).item())
+
+        gpu_usage = get_GPU_memory_usage()
```

Comment:
is it possible to lazy compute so we only run this when the metric logs are fired?